tempesta_fw/connection.h

```diff
@@ -380,7 +379,10 @@ tfw_connection_put(TfwConn *conn)
 		conn->destructor(conn);
 }
 
-#define tfw_cli_conn_put(c)	tfw_connection_put((TfwConn *)(c))
+/*
+ * Paired with tfw_srv_conn_get_if_live() via tfw_http_get_srv_conn() or
```

Comment:
The comment is misplaced. Not the `tfw_srv_conn_put()` function itself is paired with ` tfw_srv_conn_get_if_live()`. Instead commit must be placed on that place where the function is used.